core/storage/statistics/gae_models.py

```diff
@@ -482,7 +630,7 @@ class StateAnswersModel(base_models.BaseModel):
     information is duplicated across all shards, since they are immutable or are
     local to that shard.
 
-    The id/key of instances of this class has the form
+    The ID/key of instances of this class has the form
```

Comment:
End comments by a full stop. Please apply it throughout.

Reply:
done